### PR TITLE
Changelog entry for Device onboarding improvements #3256

### PR DESCRIPTION
**Pull request by Steve-Mcl**
## Description

Changelog for the addition of device quick connect

## Related Issue(s)

https://github.com/FlowFuse/flowfuse/issues/3256

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated

### CODE REVIEW

File: src/changelog/2024/02/device-onboarding-improvements.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Device Onboarding Improvements
+date: 2024-02-05 13:00:00.0
+authors: ["stephen-mclaughlin"]
+tags:
+    - changelog
+---
+
+Hot on the heals of our recent onboarding improvements and streamlining, we 
+have now introduced a new simpler and faster way of getting your devices
```
Comment (MarianRaphael) [inline src/changelog/2024/02/device-onboarding-improvements.md:10]:
```suggestion
have now introduced a new, simpler and faster way of getting your devices
```

**Review by MarianRaphael — APPROVED** [review]
@Steve-Mcl, could you also provide a link to the documentation?

**Comment by Steve-Mcl** [comment]
> @Steve-Mcl, could you also provide a link to the documentation?

Sure. 

Question: should I remove the images since there are images in the docs? (single source of truth)

**Comment by Steve-Mcl** [comment]
> could you also provide a link to the documentation

Added a link to Device Agent docs in [1056f05](https://github.com/FlowFuse/website/pull/1624/commits/1056f05642fa769bcb3151cb574deb935780878c)

If you expected something else, please be more specific.

**Review by knolleary — APPROVED** [review]

**Pull request merged**